buildlib/dockers-arm/centos-release.Dockerfile

```diff
@@ -0,0 +1,63 @@
+ARG CUDA_VERSION
```

Comment:
why need different compose file for ARM?

Reply:
OS and MOFED support matrix differs between hardware platforms.